plugins/proofpoint_tap/plugin.spec.yaml

```
@@ -4,7 +4,8 @@ products: [insightconnect]
 name: proofpoint_tap
 title: Proofpoint TAP
 description: Parse Proofpoint Targeted Attack Protection (TAP) alerts
-version: 3.1.1
+version: 3.1.2
+supported_versions: ["v2"]
```

This is a SaaS service that doesn't have versions

```suggestion
supported_versions: ["Proofpoint TAP API v2", "Tested on 2021-8-30"]
```